model-optimizer/mo/utils/graph.py

```diff
@@ -12,15 +12,18 @@
 from mo.utils.utils import refer_to_faq_msg
 
 
-def backward_bfs_for_operation(start_node: Node, op_names: list):
+def backward_bfs_for_operation(start_node: Node, op_names: list, skip_op_list: list = None):
     """
     Find node with 'op' attribute equal to one of from 'op_name', searching in the backward direction.
     In case of branching algorithm goes into each branch, but if it can't find layer in one of them it returns
     empty list.
 
     :param start_node: Start node for BFS algorithm
     :param op_names: The list with names of operations to search
+    :param skip_op_list: list of operations to be stopped at if they are met
     """
+    if skip_op_list is None:
+        skip_op_list = []
```

Comment:
may be it is better to assign `[]` for `skip_op_list` as a default value in this function signature:
`def backward_bfs_for_operation(start_node: Node, op_names: list, skip_op_list: list = []):`

So there is no need in this `if`

Reply:
Actually, approach with default list argument has interesting side-effect which I want to avoid. Refer to https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments